docs/action_kit.rst

```diff
@@ -5,19 +5,53 @@ ActionKit
 Overview
 ********
 
-The ActionKit class allows you to interact with an `ActionKit <https://actionkit.com/>`_.
+`ActionKit <https://actionkit.com/>`_ is a platform for advocacy, fundraising, and
+get-out-the-vote. This parsons integration with the
+`ActionKit REST API <https://roboticdogs.actionkit.com/docs/manual/api/rest/overview.html>`_
+supports fetching, creating, and updating records of campaigns, events, and users.
+Bulk upload of new users and user updates is also supported.
+
+.. note::
+  Authentication
+    ActionKit requires `HTTP Basic Auth <https://en.wikipedia.org/wiki/Basic_access_authentication>`_.
+    Clients with an ActionKit account can obtain the domain, username, and password needed
+    to access the ActionKit API. See the `ActionKit REST API Authentication <https://roboticdogs.actionkit.com/docs/manual/api/rest/overview.html#authentication>`_
+    documentation for more information on obtaining ActionKit API credentials.
 
 **********
 Quickstart
 **********
 
-**Retrieve a User**
+To instantiate the ActionKit class, you can either store your ActionKit API
+domain, username, and password as environmental variables (ACTION_KIT_DOMAIN,
```

Comment:
Another minor thing.. could you wrap the environment variables' names in double backticks (e.g. \``ACTION_KIT_DOMAIN\``) so they are formatted as code?